curvefs/src/metaserver/inode_storage.h

```diff
@@ -76,6 +76,16 @@ class InodeStorage {
      */
     MetaStatusCode Insert(const Inode& inode, int64_t logIndex);
 
+    /**
+     * @brief update deleting inode key in storage
+     * @param[in] inode: the inode want to update
+     * @param[in] logIndex: the index of raft log
+     * @return
+     */
+    MetaStatusCode UpdateDeletingKey(const Inode& inode, int64_t logIndex);
```

Comment:
`UpdateDeletingKey` to `AddDeletedInode` maybe more clearer

Reply:
fix